### PR TITLE
Isaac to mujoco loading

### PR DESCRIPTION
**Pull request by is2ac2**
#28 
- added .xml, mujoco support
- changed stiffness and damping to make simulation with only legs more realistic
- setup loading model and deploying in mujoco

### CODE REVIEW

File: sim/deploy/run.py [hunk 1]
```diff
@@ -62,11 +63,18 @@ class MujocoWorld(World):
...
         self.model.opt.timestep = self.cfg.dt
+
         # First step
         self.data = mujoco.MjData(self.model)
+        self.data.qpos = self.model.keyframe("default").qpos
+
         mujoco.mj_step(self.model, self.data)
 
+        # Set self.data initial state to the default standing position
+        # self.data.qpos = StompyFixed.default_standing()
```
Comment (budzianowski) [inline sim/deploy/run.py:76]: Put TODO flag - the keyframe and default_standing should produce the same results soon

File: sim/scripts/create_fixed_torso.py [hunk 2]
```diff
@@ -3,19 +3,18 @@
 
 import xml.etree.ElementTree as ET
 
-from sim.stompy.joints import StompyFixed
+from sim.hexmove.joints import Stompy
 
-STOMPY_URDF = "sim/stompy/robot.urdf"
+STOMPY_URDF = "sim/hexmove/robot.urdf"
```
Comment (budzianowski) [inline sim/scripts/create_fixed_torso.py:8]: change back

File: sim/stompy_legs/joints.py [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+            Stompy.legs.left.knee_pitch: 2.05,
+            Stompy.legs.left.ankle_pitch: 0.33,
+            Stompy.legs.left.ankle_roll: 1.73,
+            Stompy.legs.right.hip_pitch: 0,
+            Stompy.legs.right.hip_roll: -1.6,
+            Stompy.legs.right.hip_yaw: -2.15,
+            Stompy.legs.right.knee_pitch: 2.16,
+            Stompy.legs.right.ankle_pitch: 0.5,
+            Stompy.legs.right.ankle_roll: 1.72,
+            # legs - squat from urdf
```
Comment (budzianowski) [inline sim/stompy_legs/joints.py:101]: remove below

File: sim/deploy/policy.py [hunk 4]
```diff
@@ -35,7 +35,10 @@ def __init__(
         self.hist_obs = deque()
         for _ in range(frame_stack):
             self.hist_obs.append(np.zeros([1, obs_size], dtype=np.double))
-        self.model = torch.jit.load(model_path)
+        # Load the model.
```
Comment (budzianowski) [inline sim/deploy/policy.py:38]: Put todo to bring back jit once we will be ready

File: sim/deploy/config.py [hunk 5]
```diff
@@ -57,7 +57,7 @@ class RobotConfig:
     kds: np.ndarray
     tau_limit: np.ndarray
     robot_model_path: str
-    dt: float = 0.001
+    dt: float = 0.0001  # 0.001
```
Comment (budzianowski) [inline sim/deploy/config.py:60]: why do we change this?

**Review by budzianowski — COMMENTED** [review]
Remove hexmove files. After that lgtm!

File: sim/deploy/config.py (hunk 5 above)
Comment (is2ac2) [reply sim/deploy/config.py:60]: It slows the video down so we can watch it at a reasonable speed

**Review by budzianowski — APPROVED** [review]
lgtm

**Pull request merged**